packages/core/package.json

```diff
@@ -3653,6 +3653,12 @@
                 "key": "ctrl+shift+v",
                 "mac": "cmd+shift+v",
                 "when": "editorTextFocus && editorLangId == asl || editorTextFocus && editorLangId == asl-yaml"
+            },
+            {
+                "command": "noop",
```

Comment:
Makes sense. Is there any way to intercept the save command on the custom editor using something like
```
webView.webview.onDidRecieveMessage(e => {
// whatever save does here
})
```

Reply:
I belive you mean undo in your comment, rather than save.

The command is executed inside WFS and only affects WFS internal definition, the webview does not receive any message for it.
With the suppression logic I introduced, it should work like this: 
1. User triggers undo in WFS (from their keyboard or the button in the UI)
2. WFS performs its internal undo and updates its internal definition to the previous one
3. Updated WFS internal definition is synced with the local file
4. At the same time, VSCode built-in undo is suppressed for Ctrl+Z to not interfere with the above behaviour

I did not find any better suppression options since the custom editor (including webview) has all the standard VSCode keybindings working (such as Ctrl+Z) unless we override them

Reply:
Ah interesting, I think that's fine then

Reply:
Let me check with my team to see if they have any ideas how to do this without a noop command